packages/http-client-python/generator/pygen/codegen/models/paging_operation.py

```diff
@@ -58,6 +58,15 @@ def __init__(
         self.override_success_response_to_200 = override_success_response_to_200
         self.pager_sync: str = yaml_data.get("pagerSync") or f"{self.code_model.core_library}.paging.ItemPaged"
         self.pager_async: str = yaml_data.get("pagerAsync") or f"{self.code_model.core_library}.paging.AsyncItemPaged"
+        self.continuation_token: Dict[str, Any] = yaml_data.get("continuationToken", {})
+
+    @property
+    def has_continuation_token(self) -> bool:
+        return bool(self.continuation_token.get("input") and self.continuation_token.get("output"))
+
+    @property
+    def next_variable_name(self) -> str:
```

Comment:
should we just switch our internal handling of the `next_link` property over to `_continuation_token` so it stays consistent?

Reply:
There is still some difference between next_link pattern and continuation token. Compared with next-link, continuaiton token could always call builder function to call http request.